packages/react-devtools-shared/src/bridge.js

```diff
@@ -171,13 +172,7 @@ type NativeStyleEditor_SetValueParams = {
   value: string,
 };
 
-type UpdateConsolePatchSettingsParams = {
-  appendComponentStack: boolean,
-  breakOnConsoleErrors: boolean,
-  showInlineWarningsAndErrors: boolean,
-  hideConsoleLogsInStrictMode: boolean,
-  browserTheme: BrowserTheme,
-};
+type UpdateConsolePatchSettingsParams = ConsolePatchSettings;
```

Comment:
Can you just use `ConsolePatchSettings` here instead of `UpdateConsolePatchSettingsParams`?

Reply:
sounds good, will fix

Reply:
yeah, fixed